gemrb/plugins/SDLVideo/SDL20Video.cpp

```diff
@@ -58,29 +58,26 @@ int SDL20VideoDriver::CreateDriverDisplay(const Size& s, int bpp, const char* ti
 	Log(MESSAGE, "SDL 2 Driver", "Creating display");
 	// TODO: scale methods can be nearest or linear, and should be settable in config
 	SDL_SetHint(SDL_HINT_RENDER_SCALE_QUALITY, "nearest");
-	//SDL_SetHint(SDL_HINT_RENDER_VSYNC, "0");
 
-#if OPENGL_BACKEND
-#if OPENGL_BACKEND == OpenGL
+#if USE_OPENGL_BACKEND
+#if USE_OPENGL_API
 	const char* driverName = "opengl";
-#elif
+#elif USE_GLES_API
 	const char* driverName = "opengles2";
 #endif
 	SDL_SetHint(SDL_HINT_RENDER_DRIVER, driverName);
-
-	/*
-	 these have no effect on the context created by SDL_CreateRenderer
-	   SDL_GL_SetAttribute(SDL_GL_CONTEXT_FLAGS, SDL_GL_CONTEXT_FORWARD_COMPATIBLE_FLAG);
-	   SDL_GL_SetAttribute(SDL_GL_CONTEXT_PROFILE_MASK, SDL_GL_CONTEXT_PROFILE_CORE);
-	   SDL_GL_SetAttribute(SDL_GL_CONTEXT_MAJOR_VERSION, 3);
-	   SDL_GL_SetAttribute(SDL_GL_CONTEXT_MINOR_VERSION, 2);
-	*/
 #endif
 
-#if SDL_VERSION_ATLEAST(2, 0, 10)
-	SDL_SetHint(SDL_HINT_RENDER_BATCHING, "1");
-#endif
+	if ( sdl2_version >= 2010) {
```

Comment:
Shouldn't this be sdl2_runtime_version? But neither appear set anywhere.

Reply:
FindOpenGLES2 is missing as well.

Reply:
Yes, this is an error. I added it, but didn't initialize it (and the variable is wrong). I'll fix it.

Reply:
Ok, fixed.